data-generator/src/main/java/cloud/project/datagenerator/transports/TransportsDataGenerator.java

```diff
@@ -113,14 +113,54 @@ public void updateTransportDataInTransportModules(DataUpdateType updateType, Tra
 
         String transportUpdateRequestJson = JsonConverter.convert(transportUpdateRequest);
 
-        System.out.println(updateType + " - Transport: " + transportUpdateRequestJson);
+//        System.out.println(updateType + " - Transport: " + transportUpdateRequestJson);
```

Comment:
```suggestion
```

Reply:
Gadałem o tym z Ziemcem i wniosek z rozmowy był taki, że najwygodniej by było wybierać transporty, które nie mają żadnej rezerwacji. W opisie PRa o tym też wspomniałem, że specjalnie usunąłem tworzenie rezerwacji transportów w Bootrstrapie. Chcieliśmy uniknąć jakichś problemów z tym, że komuś kto już zrobił rezerwacje zmienia się transport. Chociaż teraz jak nad tym myślę, to cena rezerwacji jest zapisana już w obiekcie rezerwacji i przy updatecie transportu jego ID zostaje takie samo, wiec w sumie to nie wiem teraz czy to usuwanie robienia rezerwacji było okej pomysłem. Może @Mixss się wypowie?

Reply:
Okej, zmienione w ostatnich commitach